tutorials/yes-no-game/full game/yesnogame.edn

```diff
@@ -0,0 +1,251 @@
+; SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Consider renaming the folder from `full game`  to `full-game` to avoid having a white-space which can cause issues in some environment.